src/test/java/software/amazon/nio/spi/s3/S3ClientStoreTest.java

```diff
@@ -226,15 +222,16 @@ public void testGenerateAsyncClientWith403Then301ResponsesNoHeader(){
     }
 
     @Test
+    // TODO: move to S3FileSystemProvider
     public void testCaching() {
-        S3Client client = S3Client.create();
+        S3Client client = S3Client.builder().region(Region.US_EAST_1).build();
```

Comment:
Isn't this the same as `S3Client.create()`?

Reply:
Why is the region needed?

Reply:
In my environment, S3Client.create() throws the following exception:

```
software.amazon.awssdk.core.exception.SdkClientException: Unable to load region from any of the providers in the 
chain software.amazon.awssdk.regions.providers.DefaultAwsRegionProviderChain@721d5b74: [software.amazon.awssdk.regions.providers.SystemSettingsRegionProvider@2b73bd6b: Unable to load region from system settings. Region must be specified either via environment variable (AWS_REGION) or  system property (aws.region)., software.amazon.awssdk.regions.providers.AwsProfileRegionProvider@673e76b3: No region provided in profile: default, software.amazon.awssdk.regions.providers.InstanceProfileRegionProvider@1f66d8e1: Unable to contact EC2 metadata service.]
	at software.amazon.awssdk.core.exception.SdkClientException$BuilderImpl.build(SdkClientException.java:111)
	at software.amazon.awssdk.regions.providers.AwsRegionProviderChain.getRegion(AwsRegionProviderChain.java:70)
	at software.amazon.awssdk.awscore.client.builder.AwsDefaultClientBuilder.regionFromDefaultProvider(AwsDefaultClientBuilder.java:282)
```
I assumed it is because not using AWS I do not have AWS_REGION set. I replaced with the builder since I noticed is used everywhere else.

Reply:
Makes sense to add a region otherwise we are relying on the test environment having a profile or a env var.